src/Command/CommandBase.php

```diff
@@ -1170,16 +1170,16 @@ protected function checkForNewVersion(InputInterface $input, OutputInterface $ou
     if (strpos($input->getArgument('command'), 'api:') !== FALSE) {
       return;
     }
-    // Bale for development builds.
+    // Bail for development builds.
     if ($this->getApplication()->getVersion() == '@package_version@') {
       return;
     }
-    // Bale in Cloud IDEs to avoid hitting Github API rate limits.
-    if (!AcquiaDrupalEnvironmentDetector::isAhIdeEnv()) {
+    // Bail in Cloud IDEs to avoid hitting Github API rate limits.
+    if (AcquiaDrupalEnvironmentDetector::isAhIdeEnv()) {
       return;
     }
     try {
-      if ($this->hasUpdate()) {
+      if (@$this->hasUpdate()) {
```

Comment:
I don't feel good about this error suppression. We already have so much trouble debugging failed update checks.

Is there a way to tie this to verbosity? So we can at least see errors in verbose output.

Reply:
Not that I know of. The problem is that you can't catch some php error and warnings.